app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(
+      text: params['comment']['text'],
+      author_id: params[:user_id],
+      post_id: params[:post_id]
+    )
```

Comment:
- [ ] Let's ensure that comment is created on behalf of the currently logged in user (current_user) of the app.

```suggestion
    @comment = Comment.new(
      text: params['comment']['text'],
      author_id: current_user.id,
      post_id: params[:post_id]
    )
```

Reply:
This issue is well resolved! ✔️